etc/profile-a-l/discord-ptb.profile

```diff
@@ -0,0 +1,17 @@
+# Firejail profile for discord-ptb   
+# This file is overwritten after every install/update
+# Persistent local customizations
+include discord-ptb.local   
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/discordptb   
```

Comment:
Never blacklisted, add a blacklist to disable-programs.

Reply:
I am not sure what you mean by this. To me it looks the same as https://github.com/netblue30/firejail/blob/master/etc/profile-a-l/discord-canary.profile but for ptb

Reply:
But there is no blacklist of it unlike for discordcanary:
https://github.com/netblue30/firejail/blob/ddb685c1ec66f9164e0721b0d8bb5ccb1a75e22d/etc/inc/disable-programs.inc#L414